ddl/table.go

```diff
@@ -14,13 +14,16 @@
 package ddl
```

Comment:
The privileges of so many files are changed.

Reply:
I don't change anything